tests/python/frontend/pytorch/test_lstms.py

```diff
@@ -0,0 +1,378 @@
+                        layer_num=model_num_layers,
+                        proj_size=projection_size,
+                        rnd_weights_init=rnd_weights,
+                        use_bias=use_bias,
+                    )
+                    hidden_layers_num = model_num_layers
+                elif lstm_type == "bp":
+                    model = LSTM_Model(
+                        device,
+                        batch_first=batch_first,
+                        bidirectional=True,
+                        proj_size=projection_size,
+                        rnd_weights_init=rnd_weights,
+                        use_bias=use_bias,
+                    )
+                    hidden_layers_num = 2
+                elif lstm_type == "sbp":
+                    model = LSTM_Model(
+                        device,
+                        batch_first=batch_first,
+                        bidirectional=True,
+                        layer_num=model_num_layers,
+                        proj_size=projection_size,
+                        rnd_weights_init=rnd_weights,
+                        use_bias=use_bias,
+                    )
+                    hidden_layers_num = 2 * model_num_layers
+                else:
+                    print("WARNING: LSTM type {} is not supported here!".format(lstm_type))
+                    return
+
+                model.eval()
+
+                # Get golden output from original model
+                input_hidden_shape = (hidden_layers_num, batch_size, model_hidden_size)
+                input_hidden_shape_with_proj = (hidden_layers_num, batch_size, projection_size)
+                dummy_input, input_shape = model.get_dummy_input()
+                golden_output_batch = model.forward(dummy_input.to(device)).detach().cpu().numpy()
+
+                dtype = "float32"
+                h_zeros = np.zeros(input_hidden_shape, dtype=dtype)
+                if has_proj:
+                    h_zeros = np.zeros(input_hidden_shape_with_proj, dtype=dtype)
+                c_zeros = np.zeros(input_hidden_shape, dtype=dtype)
+
+                tvm_output = None
+                for format in ("ts", "onnx"):
+                    if format == "ts":
+                        # Use torch.jit.trace to generate a torch.jit.ScriptModule via tracing.
+                        traced_script_module = torch.jit.trace(model, dummy_input).eval()
+
+                        # Import model to Relay
+                        shape_list = [("input", input_shape)]
+                        mod, params = relay.frontend.from_pytorch(traced_script_module, shape_list)
+
+                        # Model compilation by tvm
+                        target = tvm.target.Target("llvm", host="llvm")
+                        dev = tvm.cpu(0)
+                        with tvm.transform.PassContext(opt_level=3):
+                            lib = relay.build(mod, target=target, params=params)
+                    elif format == "onnx":
```

Comment:
Remove onnx test

Reply:
My idea to check modifications of LSTM layer converted from pytorch to onnx is related to that pytorch frontend supports less ML layers than onnx one. It means that complicated neural network contained LSTM can be not supported from pytorch model on tvm side, but onnx model converted from pytorch can be. Moreover there is specific of stacked LSTM support, I do not think that it is tested on onnx frontend side

Reply:
Ok. Then please address my other comments. Also you can avoid creating a temp directory by writing onnx file to an in-memory buffer, see https://github.com/facebookresearch/detr/blob/master/test_all.py#L136-L140

Reply:
I does not work for onnx model loading, but I found correct way with io.BytesIO using. done